### PR TITLE
Ocr support schema

### PR DESCRIPTION
**Pull request by torosent**

### CODE REVIEW

File: src/template.proto [hunk 1]
```diff
@@ -283,4 +283,41 @@ message Trigger {
 message Schedule {
     // The recurrence period of the job, set as a cron expression
     string recurrencePeriod = 1;
+}
+
+// AnonymizeImageTemplate represents the template definition of the Anonymize service for anonymizying the sensitive data
```
Comment (Ilanak) [inline src/template.proto]: change the comment..

File: src/ocr.proto [hunk 2]
```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+import "common.proto";
+
+package types;
+
+// AnonymizeRequest represents the request to the anonymize service via GRPC
```
Comment (Ilanak) [inline src/ocr.proto]: OcrRequest

File: src/ocr.proto [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+package types;
+
+// AnonymizeRequest represents the request to the anonymize service via GRPC 
+message OcrRequest {
+    // The image to ocr
+    Image image = 1;
+}
+  
+  // AnonymizeResponse represents the anonymize service response
```
Comment (Ilanak) [inline src/ocr.proto]: OcrResponse

File: src/ocr.proto [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    Image image = 1;
+}
+  
+  // AnonymizeResponse represents the anonymize service response 
+message OcrResponse {
+    // The image detected bboxes
+    Image image = 1;
+}
+  
+// The Anonymize Service is a service that anonymizes a given the text using predefined analyzers fields and anonymize configurations.
```
Comment (Ilanak) [inline src/ocr.proto]: OcrService

File: src/common.proto [hunk 5]
```diff
@@ -52,4 +52,41 @@ message Location {
...
+  // The image type JPEG/PNG/TIFF
+  string imageType = 2;
+  //Array of Boundingbox
+  repeated Boundingbox Boundingboxes = 3;
+  //Image Text
+  string text = 4;
+}
+
+// Define an extracted bounding box text/image from images
+message Boundingbox {
```
Comment (Ilanak) [inline src/common.proto]: BoundingBox?

Comment (torosent) [reply src/common.proto]: OCR output is bounding boxes of words

**Pull request merged**